data/shubhsharma19.json

```
@@ -42,32 +43,37 @@
       "name": "Instagram: I post Programming Content",
       "url": "https://instagram.com/code.shubh",
       "icon": "FaInstagram"
+    },
+     {
+      "name": "Support my work",
+      "url": "https://ko-fi.com/shubhsharma19",
+      "icon": "FaBuymeacoffee"
     }
```

Seems there's a wrong indentation here.
```suggestion
    {
        "name": "Support my work",
        "url": "https://ko-fi.com/shubhsharma19",
        "icon": "FaBuymeacoffee"
    }
```